### ISSUE TITLE
Review

### ISSUE THREAD

**Issue by slabkodima90** [oleksandr-yehorov/theyalow#1]
https://github.com/oleksandr-yehorov/theyalow/blob/e1974a8ea77224c1f4b2241ff3def00f791c0cc1/index.html#L13
- излишний контейнер + ниже коду есть моменты

https://github.com/oleksandr-yehorov/theyalow/blob/e1974a8ea77224c1f4b2241ff3def00f791c0cc1/index.html#L15
- использование тэга header и footer
- название классов немного короче, без повторений родительского

Предварительная оценка: 75

**Comment by oleksandr-yehorov** [comment]
Спасибо за замечания. Учту в следующих заданиях.

**Comment by slabkodima90** [comment]
Да, и еще, css свойства лучше прописывать в алфавитном порядке, чтобы была лучше читаемость
